### PR TITLE
Initial improvements to Overviews

### PR DESCRIPTION
**Pull request by tbedford**
## Description

This PR implements some initial tidy up of the three Overview topics in Client SDK docs. 

One aspect not present is the auto-listing of tutorials, as this requires a fix to NDP. 

## Review

[Review app](https://nexmo-developer-pr-2239.herokuapp.com/client-sdk/overview)

### CODE REVIEW

File: _documentation/en/client-sdk/overview.md [hunk 1]
```diff
@@ -1,53 +1,71 @@
...
 
 # Overview
 
-Nexmo Client SDK is a conversation-centric product consisting of iOS, Android, and JavaScript SDKs.
+Nexmo Client SDK allows you to build Programmable Conversation applications.
 
-Client SDK enables communications across multiple Channels including:
+Using the Conversation API and the Client SDK you can build complete applications that feature two-way voice and messaging communications. Whereas the [Conversation API](/conversation/overview) can be used directly to build out the backend of your Programmable Conversation application, the Client SDK enables to to quickly and easily build the client-side application. The backend usually deals with tasks such as user database management, the generation of user tokens, and the creation of the [Conversations](/conversation/concepts/conversation). The client-side provides methods to log users into Nexmo, and handle the various [Events](/conversation/concepts/event) that are generated as communication takes place. Other functions of the client-side include generating member invites if required, handling typing indicators, and handling voice call operations such as inbound call, making a call, a user hanging up, and so on.
```
Comment (marklewin) [inline _documentation/en/client-sdk/overview.md]:
```suggestion
Using the Conversation API and the Client SDK you can build complete applications that feature two-way voice and messaging communications. Whereas the [Conversation API](/conversation/overview) can be used directly to build out the backend of your Programmable Conversation application, the Client SDK enables you to quickly and easily build the client-side application. The backend usually deals with tasks such as managing users in a database, the generation of tokens for those users, and the creation of the [Conversations](/conversation/concepts/conversation). The client-side provides methods to log users into Nexmo, and handle the various [Events](/conversation/concepts/event) that are generated as communication takes place. Other client-side functions include generating member invites if required, displaying typing indicators, and handling voice call operations such as inbound call, making a call, a user hanging up, and so on.
```

File: _documentation/en/client-sdk/overview.md [hunk 2]
```diff
@@ -1,53 +1,71 @@
...
-## Next Steps
-
 Read more about:
 
-* [In-App Messaging](/client-sdk/in-app-messaging/overview)
 * [In-App Voice](/client-sdk/in-app-voice/overview)
 
+## Concepts
+
+Many of the concepts used in the Client SDK are a result of the design of the Conversation API, which is the underlying technology. Concepts that will be useful in using the Client SDK are as follows:
```
Comment (marklewin) [inline _documentation/en/client-sdk/overview.md]:
```suggestion
Many of the concepts used in the Client SDK are a result of the design of the [Conversation API](/conversation/overview), which is the underlying technology. The following Concepts will help you use the Client SDK:
```

File: _documentation/en/client-sdk/in-app-messaging/overview.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Overview
+description: This topic provides an overview of using the Client SDK to build in-app messaging applications.
 ---
 
 # In-App Messaging Overview
 
-Nexmo In-App Messaging enables you to build chat experience as part of your mobile and web app over websockets. The Nexmo Client SDKs offers functionality such as:
+Client SDK enables you to build chat experience as part of your mobile and web app over websockets.
```
Comment (marklewin) [inline _documentation/en/client-sdk/in-app-messaging/overview.md]: Should this be "chat experiences" or "a chat experience"?

Comment (tbedford) [reply _documentation/en/client-sdk/in-app-messaging/overview.md]: Yes, horrible wording. I've changed it.

**Pull request merged**